patches/build-config-win-BUILD.gn.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/build/config/win/BUILD.gn b/build/config/win/BUILD.gn
+index c51ad032acd3f6efe98527db0b3d104ca9b4740d..05739cf280d2042de8fe5ed9437d52cfbe0dbc9e 100644
+--- a/build/config/win/BUILD.gn
++++ b/build/config/win/BUILD.gn
+@@ -118,7 +118,7 @@ config("compiler") {
+         cflags += [ "--target=x86_64-windows" ]
+       }
+     } else if (current_cpu == "arm64") {
+-      cflags += [ "--target=arm64-windows" ]
++      cflags += [ "--target=aarch64-pc-windows" ]
```

Comment:
have you looked into setting gn arg `fatal_linker_warnings = false` for windows arm64 builds?

we can do this while we don't have goma support for this triple.

Reply:
It's not goma that is the issue, it's a bug in the windows linker. This is actually the change that breaks goma, but it's necessary to work around the windows linker bug.

Reply:
I also need to pull this out into the brave patches commit to separate it from upstream changes